Nautilus/Handlers/ModDataBankHandler.cs

```diff
@@ -43,10 +37,9 @@ private static void CompleteRegister(PDAEncyclopedia.EntryData data)
     /// Register mod with default values. 
     /// </summary>
     /// <param name="assembly">Assembly for your mod.</param>
-    public static void RegisterMod(Assembly assembly)
+    public static void RegisterMod(string GUID)
     {
-        var bepinplugin = assembly.GetTypes().First(type => type.GetCustomAttributes(false).OfType<BepInPlugin>().Any());
-        var bepinplugindata = bepinplugin.GetCustomAttributes(false).OfType<BepInPlugin>().FirstOrDefault();
+        var bepinplugindata = BepInEx.Bootstrap.Chainloader.PluginInfos.FirstOrDefault(info => info.Value.Metadata.GUID == GUID).Value.Metadata;
```

Comment:
`BepInEx.Bootstrap.Chainloader.PluginInfos[GUID]`? Or `.TryGetValue` if you wanna be extra safe. No need to iterate a dictionary as it's meant to do lookups

Reply:
Probably should use the TryGetValue way tbh

Reply:
Ah alright.

Reply:
Should be fixed.